ios/Classes/MobileScanner.swift

```diff
@@ -48,6 +48,12 @@ public class MobileScanner: NSObject, AVCaptureVideoDataOutputSampleBufferDelega
 
     var detectionSpeed: DetectionSpeed = DetectionSpeed.noDuplicates
 
+    var shouldConsiderInvertedImages: Bool = false
+    // local variable to invert this image only this time,
```

Comment:
I think this can be a one line comment.

```suggestion
    // This flag is used to determine if the current frame should be color-inverted.
```